### DISCUSSION TITLE
Repush changes

### DISCUSSION THREAD

**Discussion by Zeturios** [opengisch/QField#3345] [Q&A]
Hello fellow Qfielders,

I recently experienced a problem where I pushed the changes from my Qfield device - the app stated it was succesfully - and afterwards didn't find them in my freshly synced QGis Project or in the Qfield Cloud Project in the Wep as Deltas. 
For me there are now two options:
1. Trying to Sync the Project (with all the data getting lost maybe)
2. Getting access to the local copy of the cloud project on my android device.

But for the second Intent I don't find any information on how to access the Cloud Projects on the android device.

It would be real blast if anyone could help!

With kindest regards!

Zeturios

**Comment by suricactus** [comment]
Hi @Zeturios did you check if you see your changes on QFieldCloud -> Project -> Changes? You can check their status and you can mark them to be applied again.

![image](https://user-images.githubusercontent.com/2820439/190588110-9f281fda-0ea9-4ad5-bcb6-14aba763299e.png)

Also, if you need to access you files, just view them easily from the menu:

![image](https://user-images.githubusercontent.com/2820439/190587849-77597788-5086-493c-8f77-6d58eddca9f1.png)
